odxtools/diaglayers/diaglayer.py

```diff
@@ -124,7 +124,7 @@ def _resolve_odxlinks(self, odxlinks: OdxLinkDatabase) -> None:
             # imported references only apply within this specific
             # diagnostic layer
             extended_odxlinks = copy(odxlinks)
-            extended_odxlinks.update(imported_links)
+            extended_odxlinks.update(imported_links, False)
```

Comment:
```suggestion
            extended_odxlinks.update(imported_links, overwrite=False)
```

Reply:
I added it at the same time, thank you @kayoub5 .